README.md

```diff
@@ -289,6 +289,13 @@ $ particle flash --usb firmware.bin
 
   eg. `particle compile photon xxx` OR `particle compile p xxxx` both targets the photon
 
+  **NOTE**: There is also a `defaultCompileDevice` flag in `settings.js` that allows the user to target a device by default and omit the `device_type`argument.
+
+  The default target device is **core**.
+
+  eg. `particle compile .` VS `particle compile core .`
+
+
```

Comment:
I will have to move that flag to the `.json` file instead since it was introduced in the `settings.js`